doc/htmldoc/installation/cmake_options.rst

```diff
@@ -107,7 +107,7 @@ Generic build configuration
 | ``-Dstatic-libraries=[OFF|ON]``                      | Build static executable and libraries [default=OFF].             |
 +------------------------------------------------------+------------------------------------------------------------------+
 | ``-Dwith-optimize=[OFF|ON|<list;of;flags>]``         | Enable user defined optimizations                                |
-|                                                      | [default=OFF (uses '-O2')]. When ON, '-O3' is used. Separate     |
+|                                                      | [default=OFF (uses '-O0')]. When ON, '-O3' is used. Separate     |
```

Comment:
Adjust as above. I struggle adding that as a proper suggestion here since the text is split over modified and unmodified lines.